mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -139,8 +139,8 @@ def __init__(self, name=None, bus=None, use_settings=True):
         self.settings = None
         self.settings_write_path = None
 
-        if use_settings:
-            self._init_settings()
+        # TODO deprecate, old kludge from when padatious was a fallback skill
+        self._use_settings = use_settings
```

Comment:
how TODO? if a warning can go up now, the deprecation clock can start immediately. there's no harm in taking *more* than X months/Y versions.

Reply:
in fact it seems to be completely unused already, marked for removal in 0.0.3